### ISSUE TITLE
 Приведение строки "" к типу "Integer" является недопустимым.

### ISSUE THREAD

**Issue by adastmin1** [dmitryvhf/L2ScriptMaker#21]
![image](https://user-images.githubusercontent.com/60781118/74028967-a9d1ce00-49bc-11ea-8c3b-a98ba07c1409.png)

************* Текст исключения **************
System.InvalidCastException: Приведение строки "" к типу "Integer" является недопустимым. ---> System.FormatException: Входная строка имела неверный формат.
   в Microsoft.VisualBasic.CompilerServices.Conversions.ParseDouble(String Value, NumberFormatInfo NumberFormat)
   в Microsoft.VisualBasic.CompilerServices.Conversions.ToInteger(String Value)
   --- Конец трассировки внутреннего стека исключений ---
   в Microsoft.VisualBasic.CompilerServices.Conversions.ToInteger(String Value)
   в L2ScriptMaker.ItemDataGenerator.StartButton_Click(Object sender, EventArgs e)
   в System.Windows.Forms.Control.OnClick(EventArgs e)
   в System.Windows.Forms.Button.OnMouseUp(MouseEventArgs mevent)
   в System.Windows.Forms.Control.WmMouseUp(Message& m, MouseButtons button, Int32 clicks)
   в System.Windows.Forms.Control.WndProc(Message& m)
   в System.Windows.Forms.ButtonBase.WndProc(Message& m)
   в System.Windows.Forms.Button.WndProc(Message& m)
   в System.Windows.Forms.Control.ControlNativeWindow.WndProc(Message& m)
   в System.Windows.Forms.NativeWindow.Callback(IntPtr hWnd, Int32 msg, IntPtr wparam, IntPtr lparam)

**Comment by adastmin1** [comment]
[itemdatagen_dat_example.zip](https://github.com/dmitryvhf/L2ScriptMaker/files/4170866/itemdatagen_dat_example.zip)

**Comment by dmitryvhf** [comment]
Problem found.
This generator waiting itemdata-e.txt with **cnt0** value inside. This use for armor sets collector. But... only CT25 itemdata-e.txt have it.
I cant do universal generator for all chronicles yet. Now, i need update documentation; show information about chronicles script version.

**Comment by drdeft** [comment]
Yes, we too have this problem. What can we do for you to fix it as fast as possible, dmitryvhf?

**Comment by dmitryvhf** [comment]
> Yes, we too have this problem. What can we do for you to fix it as fast as possible, dmitryvhf?
I need fully recode this module. I need to do it more universal. Now this support CT15 chronicles only and with "fixed" dat file scructure.
I'll plan this to refactor with high priority level.
